scripts/Muon/GUI/Common/plot_widget/plot_widget_view_interface.py

```diff
@@ -17,6 +17,16 @@ class PlottingWidgetViewMeta(type(QtWidgets.QWidget), ABCMeta):
 
 class PlotWidgetViewInterface(metaclass=PlottingWidgetViewMeta):
 
+    @abstractmethod
+    def show_plot_diff(self):
+        #Shows Plot Difference Checkbox
```

Comment:
These should be in the doc string format using Triple quotes.

See the other functions in this file